cli/azd/pkg/auth/cache.go

```diff
@@ -52,6 +93,34 @@ func (a *msalCacheAdapter) Export(ctx context.Context, cache cache.Marshaler, _
 	return a.cache.Set(cCurrentUserCacheKey, val)
 }
 
+// Normalize keys by removing upper-case keys and replacing them with lower-case keys.
+// In the case where a lower-case key and upper-case key exists, the lower-case key entry
+// takes precedence.
+func normalizeKeys(m map[string]json.RawMessage) {
+	for k, v := range m {
```

Comment:
nit. Use more descriptive names for variables

Reply:
Do value the feedback. The reason it's written is for shorter names to reduce mental load. Scope is small enough that I think this okay.